lua/rustaceanvim/config/internal.lua

```diff
@@ -42,11 +42,11 @@ local RustaceanDefaultConfig = {
     ---@class RustaceanHoverActionsConfig
     hover_actions = {
 
-      --- whether to replace Neovim's built-in `vim.lsp.buf.hover`
+      --- whether to replace Neovim's built-in `vim.lsp.buf.hover`.
       ---@type boolean
       replace_builtin_hover = true,
 
-      -- the border that is used for the hover window or explain_error window
+      -- the border that is used for the hover window.
```

Comment:
issue: we should remove all fields that have been moved to `float_win_config` from the default `hover_actions`